content/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-files.md

```diff
@@ -5,30 +5,34 @@ weight: 300
 toc: true
 ---
 
-A **resource mapping file** is a JSON file that provides a map of a **lookup name** to a collection of **properties** about an AWS resource. Each mapping has the following format:
+A **resource mapping file** is a JSON file that provides a map of **lookup names** to a collection of **attributes** about AWS resources. O3DE runtimes can query the mapping file to easily find and utilize AWS resources. Usually different deployments, and consequently different mappings, are used for game development and for the released version of a game.
+
+A mapping file contains one or more resource entries and a set of global attributes.
 
 **Resource entry format**
 
+Each entry defines a lookup name - a logical name for the resource - and a collection of attributes in the following format:
+
 ```json
-string  {
+"<lookup name>":  {
     "Type": string, (required)
     "Name/ID": string, (required)
     "Region": string, (optional)
     "AccountId": string (optional)
 }
 ```
 
-Each entry has a lookup string - a logical name for the resource - that is used to find entries in the mapping file during usage.
-
-* Type - AWS resource type, e.g. AWS::Lambda::Function, AWS::S3::Bucket, etc.
-* Name/ID - AWS resource name or id
-* Region - Used to override global region attribute if this resource is deployed to a different AWS region.
-* AccountId - Used to override global account id attribute if we want to support cross account use case (arguable)
+***Supported attributes***
+* Type - AWS resource type, e.g. AWS::Lambda::Function, AWS::S3::Bucket etc. Should map to a [CloudFormation resource type](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-template-resource-type-ref.html).
+* Name/ID - AWS resource name or id.
+* Region - Used to override the global region attribute if this resource is deployed to a different AWS region.
```

Comment:
The field is mandatory when the global region is not set and it's used to construct the service URL.

Reply:
global region attribute is mandatory as always, customer won't be able to skip setting global region attribute

Reply:
JSON schema defines global region as required, so it must be provided for the mapping to be valid.